.github/workflows/staging.yml

```
@@ -0,0 +1,41 @@
+name: Build image (main)
+
+on:
+  push:
+    branches:
+      - main
```

`main`じゃなくて`master`ですね
```suggestion
      - master
```